### DISCUSSION TITLE
ScriptReference/Input-mousePosition

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#644] [Notes]
# ScriptReference/Input-mousePosition

https://docs.unity3d.com/ScriptReference/Input-mousePosition.html

**Comment by gnoqui** [comment]
In the unity editor the mouse position do not update if the Game window is not focus.

**Reply by gnoqui** [reply]
Event.current.mousePosition work and update correctly the position even when the game window is unfocus.
